pkg/providers/nutanix/env.go

```diff
@@ -0,0 +1,30 @@
+package nutanix
+
+import (
+	"fmt"
+	"os"
+
+	anywherev1 "github.com/aws/eks-anywhere/pkg/api/v1alpha1"
+)
+
+const (
+	nutanixUsernameKey = "NUTANIX_USER"
+	nutanixPasswordKey = "NUTANIX_PASSWORD"
+	nutanixEndpointKey = "NUTANIX_ENDPOINT"
```

Comment:
Are these well known Nutanix variables or something we've decided on for EKS-A?

Reply:
Same env vars as we use for CAPX.